docs/articles/documentation/using-testcafe/common-concepts/reporters.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: docs
+title: Reporters
+permalink: /documentation/using-testcafe/common-concepts/reporters/
+---
+# Reporters
+
+Reporters are plugins used to output test run reports in a certain format.
+
+TestCafe ships with the following reporters:
+
+* [spec](https://github.com/DevExpress/testcafe-reporter-spec) - used by default
+* [list](https://github.com/DevExpress/testcafe-reporter-list)
+* [minimal](https://github.com/DevExpress/testcafe-reporter-minimal)
+* [xUnit](https://github.com/DevExpress/testcafe-reporter-xunit)
+* [JSON](https://github.com/DevExpress/testcafe-reporter-json)
+
+You can also create a [custom repoter](/testcafe/documentation/extending-testcafe/custom-reporter-plugin/) that will fit your needs.
+
+## Installing Reporters
+
+You can install reporter packages from npm in two ways:
+
+* **locally** - if you are going to use reporters within TestCafe installed locally.
+
+    Navigate to the TestCafe directory and run `npm install testcafe-reporter-{reporterName}`.
```

Comment:
You should navigate to your project directory, there you have tests. Locally means project-scoped, not OS-scoped